public/app/features/logs/components/LogDetails.tsx

```diff
@@ -111,53 +115,71 @@ class UnThemedLogDetails extends PureComponent<Props> {
                         getStats={() => calculateLogsLabelStats(getRows(), key)}
                         onClickFilterOutLabel={onClickFilterOutLabel}
                         onClickFilterLabel={onClickFilterLabel}
+                        onClickShowField={onClickShowField}
+                        onClickHideField={onClickHideField}
                         row={row}
                         app={app}
+                        wrapLogMessage={wrapLogMessage}
+                        displayedFields={displayedFields}
                       />
                     );
                   })}
+                {fields.map((field) => {
+                  const { key, value, fieldIndex } = field;
+                  return (
+                    <LogDetailsRow
+                      key={`${key}=${value}`}
+                      parsedKey={key}
+                      parsedValue={value}
+                      onClickShowField={onClickShowField}
+                      onClickHideField={onClickHideField}
+                      onClickFilterOutLabel={onClickFilterOutLabel}
+                      onClickFilterLabel={onClickFilterLabel}
+                      getStats={() =>
+                        fieldIndex === undefined
+                          ? this.getStatsForField(key)
+                          : calculateStats(row.dataFrame.fields[fieldIndex].values.toArray())
+                      }
+                      displayedFields={displayedFields}
+                      wrapLogMessage={wrapLogMessage}
+                      row={row}
+                      app={app}
+                    />
+                  );
+                })}
 
-                {detectedFieldsAvailable && (
+                {linksAvailable && (
                   <tr>
-                    <td colSpan={5} className={style.logDetailsHeading} aria-label="Detected fields">
-                      Detected fields
-                      <Tooltip content="Fields that are parsed from log message and detected by Grafana.">
-                        <Icon
-                          name="question-circle"
-                          size="xs"
-                          className={css`
-                            margin-left: ${theme.spacing(0.5)};
-                          `}
-                        />
-                      </Tooltip>
+                    <td colSpan={100} className={style.logDetailsHeading} aria-label="Data Links">
+                      Data Links
```

Comment:
As far as I understand, we would be removing the concept of _Detected fields_ and adding _Data Links_. Would there be any chance for users to either miss _Detected fields_ and/or be confused by the concept of _Data Links_?

What I mean is that if this change would be undestandable/expected or we need to do any UX/further clarification of its meaning.

Reply:
I don't think users will be confused by _Data Links_, as those are only links (derived fields) which previously would end up in _Detected fields_. 

After some chats with @niat22 and @stevendungan we agreed on monitoring the confusion/frustration. However we could add a small note for some duration, that _Detected Fields_ can be brought back by simply adding the right parser to your query. I am not decided on that, as we already suggest parsers in the query builder.

Reply:
However we definitely shouldn't forget to change documentation https://grafana.com/docs/grafana/latest/explore/logs-integration/.

Added a FU issue: https://github.com/grafana/grafana/issues/61026